db/migrate/20170421194857_create_github_contributors.rb

```diff
@@ -0,0 +1,18 @@
+class CreateGithubContributors < ActiveRecord::Migration
+  def change
+    create_table :github_contributors do |t|
```

Comment:
It will be better to create a `github_metrics` table first and then use `metics_id` as the foreign key in `github_contributors` table. In this way, if we want to add more metrics in the future, we only need to add a new record in `github_metrics` table, instead of changing db structure.